lib/core/HeadRequestAsyncTask.js

```diff
@@ -34,6 +35,21 @@ var _execute = function (callback) {
 		self.callback(null, result);
 	};
 
+    // use http proxy if configured
+    // source from: http://stackoverflow.com/questions/3862813/how-can-i-use-an-http-proxy-with-node-js-http-client
+    var match;
+    if (process.env.http_proxy != null) {
+        match = process.env.http_proxy.match(/^(http:\/\/)?([^:\/]+)(:([0-9]+))?/i);
```

Comment:
It could be done with a parameter. But many programs use this Env variable if it is present - including the request package, npm, wget, curl and browsers.